[2023-04-21 11:31:30,686] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: first_python_dag.first_python_task manual__2023-04-21T11:31:29.544419+00:00 [queued]>
[2023-04-21 11:31:30,694] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: first_python_dag.first_python_task manual__2023-04-21T11:31:29.544419+00:00 [queued]>
[2023-04-21 11:31:30,695] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2023-04-21 11:31:30,695] {taskinstance.py:1357} INFO - Starting attempt 1 of 2
[2023-04-21 11:31:30,696] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2023-04-21 11:31:30,710] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): first_python_task> on 2023-04-21 11:31:29.544419+00:00
[2023-04-21 11:31:30,714] {standard_task_runner.py:52} INFO - Started process 2109 to run task
[2023-04-21 11:31:30,717] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'first_python_dag', 'first_python_task', 'manual__2023-04-21T11:31:29.544419+00:00', '--job-id', '19', '--raw', '--subdir', 'DAGS_FOLDER/second.py', '--cfg-path', '/tmp/tmpwktb_eii', '--error-file', '/tmp/tmpxyne1gyx']
[2023-04-21 11:31:30,717] {standard_task_runner.py:80} INFO - Job 19: Subtask first_python_task
[2023-04-21 11:31:30,765] {task_command.py:369} INFO - Running <TaskInstance: first_python_dag.first_python_task manual__2023-04-21T11:31:29.544419+00:00 [running]> on host 7b4a1553b0dc
[2023-04-21 11:31:30,826] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=first_python_dag
AIRFLOW_CTX_TASK_ID=first_python_task
AIRFLOW_CTX_EXECUTION_DATE=2023-04-21T11:31:29.544419+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-04-21T11:31:29.544419+00:00
[2023-04-21 11:31:30,827] {logging_mixin.py:115} INFO - CURRENT DATETIME IS 2023-04-21 11:31:30.827228
[2023-04-21 11:31:30,827] {python.py:173} INFO - Done. Returned value was: None
[2023-04-21 11:31:30,837] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=first_python_dag, task_id=first_python_task, execution_date=20230421T113129, start_date=20230421T113130, end_date=20230421T113130
[2023-04-21 11:31:30,889] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-04-21 11:31:30,915] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
